### ISSUE TITLE
Invalid error with Squiz.ControlStructures.InlineIfDeclaration.SpacingBeforeElse and unicode characters.

### ISSUE THREAD

**Issue by mducharme** [squizlabs/PHP_CodeSniffer#681]
The Sniff "Squiz.ControlStructures.InlineIfDeclaration.SpacingBeforeElse" (and probably others) fail to report the proper spacing when using unicode character in the inline comparison.

How to reproduce:
```php
$r = $expr ? '✓' : '';
```
The previous code should be valid (one space between all operators) but `phpcs` returns:
`Inline shorthand IF statement requires 1 space before ELSE; -1 found`

As a sidenote, the following PASSES (but should fail, there are 3 space before the `:` operator)
```php
$r = $expr ? '✓'   : '';
```

**Comment by aik099** [comment]
Have you specified `utf-8` encoding in ruleset or when invoking `phpcs` command?

**Comment by mducharme** [comment]
This did the trick!
Somehow my "encoding" rule was not in the proper XML tag...

Sorry for the noise,
